docs/release_process.md

```diff
@@ -59,19 +59,9 @@ merge back to `main` when they are ready.
 
 The standard release interval for Monitor is one week, meaning there should be at least one new version of the Monitor web app on the [Production][prod] environment. However, since we've started doing 1-click deploy to production, our release cycle can become more and more frequent.
 
-## Preview Deployment
-
-Every time a PR is opened, a docker image is created and deployed to the
-preview deployment environment powered by GCP Cloud Run / CloudSQL.
-A brand new database is created and schema is migrated specific to that PR.
-A brand new Cloud Run service is set up and cleaned up along with the
-database at the end of the lifecycle of that PR (when closed or merged).
-A preview URL is generated and linked in the PR when the environment is
-set up and changes are ready to be reviewed.
-
 ## Release to Stage
 
-Every commit to `main` is automatically deployed to the [Stage][stage] server via Github Actions and Jenkins.
+Every commit to `main` is automatically deployed to the [Stage][stage] server via Github Actions. The image is built and uploaded to Github Artifact Registry. ArgoCD scans the registry and detects whenever a new image uploaded with a tag that is a short commit SHA. When one is detected, it triggers a new deployment.
```

Comment:
Heh, and here I was thinking GAR stood for Google Artifact Repository :)

Do you happen to have a link to where Argo is configured to look for short commit SHAs in the repository, and if so, could you add it here? That would help for me to get a feel for how it all fits together.